tests/run-pass/intptrcast.rs

```diff
@@ -1,4 +1,7 @@
-// compile-flags: -Zmiri-seed=0000000000000000
+// compile-flags: -Zmiri-seed=0000000000000000 
+unsafe fn transmute_ptr_to_int<T>(x: *const T) -> usize { 
```

Comment:
No need to make it unsafe, this is a safe operation.

Reply:
Solved in the last push